tools/notification-configuration/github-codeowner-subscriber/Azure.Sdk.Tools.GithubCodeownerSubscriber.csproj

```diff
@@ -15,7 +15,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\code-owners-parser\CodeOwnersParser\Azure.Sdk.Tools.CodeOwnersParser.csproj" />
-    <ProjectReference Include="..\common\common.csproj" />
+    <ProjectReference Include="..\..\common\common.csproj" />
```

Comment:
I don't think we want to create a folder called common and put that in the root of the tools repo as that is too generic. We should either rename it to match what it really is or move it under one of the other directories and reference from there.

Reply:
Our current common folder is all related pulling identity info from kusto database. 
Proposal:
1. Rename `common` to `identity-resolver`
2. Rename `identity-resolver` to `identity-converter`, as it used for convert github and ms alias back and forth

Reply:
I think that is reasonable, but as I mentioned in my other comment lets keep these grouped together under one tools folder. Maybe we have the tools/identity-resolution/ and under it a library called identity-resolution and the exe called identity-converter.

Reply:
The latest changes:

```
-identity-resolution
   -identity-resolution(original common codes)
   -Azure.Sdk.Tools.IdentityConverter
-notification-configuration
   -github-codeowner-subscriber (have a ref to identity-resolution)
   -notification-creator(have a ref to identity-resolution)
```